frontend/pages/settings.js

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from 'react'
+
+const SettingsPage = props => {
+  // react state for these shits
+  // preload
+  const [email, setEmail] = useState('email@email.com')
+  const [username, setUsername] = useState('username1')
```

Comment:
You probably want these as empty and use placeholder attribute to add something in the input box if you want